ledfx/devices/__init__.py

```diff
@@ -163,7 +163,13 @@ def update_pixels(self, virtual_id, data):
                 # _LOGGER.debug(f"Device {self.id} flushed by Virtual {virtual_id}")
 
                 self._ledfx.events.fire_event(
-                    DeviceUpdateEvent(self.id, frame)
+                    DeviceUpdateEvent(
+                        self.id,
+                        frame,
+                        self._ledfx.virtuals._virtuals[self.id]._config[
```

Comment:
This is UGLY and I need to find a better way to get a device rows, not even sure if this is always true

Reply:
improved